src/services/whitelist/whitelist.service.js

```diff
@@ -1,20 +1,57 @@
 // Initializes the `whitelist` service on path `/whitelist`
-const createService = require('feathers-memory');
-
-const hooks = require('./whitelist.hooks');
+const { getTokenBySymbol } = require('../../utils/tokenHelper');
 
 module.exports = function whitelist() {
   const app = this;
 
-  const options = {
-    name: 'whitelist',
+  const whitelistService = {
+    async find() {
```

Comment:
whitelist has a service with all methods find, create, patch , ....
but we just need find() method so I changed this, and removed the hooks